src/components/Form.jsx

```diff
@@ -0,0 +1,23 @@
+import React, { useContext, useState } from 'react';
+import { DataContext } from '../contexts/DataContext';
+
+export const Form = () => {
+  const context = useContext(DataContext);
```

Comment:
what do you think to put directly here the 

`const { create } = useContext(DataContext);`